### PR TITLE
modified queries by using flattening tables

### PR DESCRIPTION
**Pull request by vkt1414**

### CODE REVIEW

**Comment by vkt1414** [comment]
this post shows why it may be better to use LEFT JOIN instead of CROSS JOIN while unnesting..https://www.yuichiotsuka.com/google-bigquery-unnest-arrays/
and this document also shows why CROSS JOINS can sometimes be tricky
[BigQuery_UNNEST_example_Towards_Data_Science.pdf](https://github.com/ImagingDataCommons/etl_flow/files/10785664/BigQuery_UNNEST_example_Towards_Data_Science.pdf)

**Comment by fedorov** [comment]
@bcli4d for the context, this is the work to update our derived queries to handle the attributes specific to the new collection, and at the same time optimize/simplify the queries to make those additions easier. Vamsi and I are working to finalize this today.

**Comment by fedorov** [comment]
Few other minor comments:
* make sure there not extra columns that are not used that are introduced in the output (need to confirm)
* add tab to the nested levels of the query

**Review by fedorov — CHANGES_REQUESTED** [review]

**Comment by fedorov** [comment]
@deepakri201 can you please take the queries here and do your testing to confirm they work as expected for you? Looks great to me, but I would appreciate your checks as well.

File: bq/derived_table_creation/BQ_Table_Building/derived_data_views/sql/measurement_groups.sql [hunk 1]
```diff
@@ -1,215 +1,129 @@
...
+-- We only want to include CONTAINER value types in the first level of content sequence
+AND cs_l0.ValueType IN ('CONTAINER')
+
+-- We only want to include Measurement Group Code Meanings in the second level of content sequence
+AND cs_l1_cncs.CodeMeaning IN ("Measurement Group")
+
+-- We want to include certain value types and code values in the third level of content sequence
+AND (
+  -- TEXT value type with specific Code Value and Coding Scheme Designator
+  (cs_l2.ValueType IN ('TEXT') AND cs_l2_cncs.CodeValue IN ('112039')AND cs_l2_cncs.CodingSchemeDesignator IN ('DCM'))
```
[the remark this answers is not in the document]
Comment (vkt1414) [reply bq/derived_table_creation/BQ_Table_Building/derived_data_views/sql/measurement_groups.sql]: adjusted formatting and added codemeanings

**Comment by fedorov** [comment]
@s-paquette the updated queries will result in some new values - it would be good if you checked them before we finalize this.

File: bq/derived_table_creation/BQ_Table_Building/derived_data_views/sql/measurement_groups.sql [hunk 2]
```diff
@@ -1,215 +1,124 @@
...
+
+  -- We only want to include CONTAINER value types in the first level of content sequence
+  AND cs_l0.ValueType IN ('CONTAINER')
+
+  -- We only want to include Measurement Group Code Meanings in the second level of content sequence
+  AND cs_l1_cncs.CodeMeaning IN ("Measurement Group")
+
+  -- We want to include certain value types and code values in the third level of content sequence
+  AND (
+    -- Tracking Identifier--TEXT value type with specific Code Value and Coding Scheme Designator
```
Comment (fedorov) [inline bq/derived_table_creation/BQ_Table_Building/derived_data_views/sql/measurement_groups.sql:77]: @vkt1414 what I suggested earlier is to include CodeMeaning value corresponding to the CodeValue/CodingSchemeDesignator values in the line below. So, in this particular example, CodeMeaning = "Tracking Identifier". Does it make sense? Can you add values of CodeMeaning corresponding to each combinations of CodeValue/CodingSchemeDesignator in this and other queries?

Comment (fedorov) [reply bq/derived_table_creation/BQ_Table_Building/derived_data_views/sql/measurement_groups.sql:77]: Sorry, I meant mention CodeMeaning in the line above, in the comment - not in the query itself - just to make the query more readable.

Comment (fedorov) [reply bq/derived_table_creation/BQ_Table_Building/derived_data_views/sql/measurement_groups.sql:77]: Never mind, I see I missed it. This is what happens with multitasking. It's all good!

**Review by fedorov — APPROVED** [review]

**Pull request merged**